geomesa-kafka/geomesa-kafka-datastore/src/test/scala/org/locationtech/geomesa/kafka/LiveKafkaConsumerFeatureSourceTest.scala

```diff
@@ -105,6 +109,60 @@ class LiveKafkaConsumerFeatureSourceTest extends Specification with HasEmbeddedK
         features.hasNext must beFalse
       }
     }
+
+    "support listeners" >> {
+      val m = AtomicLongMap.create[String]()
+
+      val id = "testlistener"
+      val numUpdates = 100
+      val maxLon = 80.0
+      var latestLon = -1.0
+
+      val sft = {
+        val sft = SimpleFeatureTypes.createType("listeners", "name:String,age:Int,dtg:Date,*geom:Point:srid=4326")
+        KafkaDataStoreHelper.createStreamingSFT(sft, zkPath)
+      }
+      val producerDS = DataStoreFinder.getDataStore(producerParams)
+      producerDS.createSchema(sft)
+
+      val listenerConsumerDS = DataStoreFinder.getDataStore(consumerParams)
+      val consumerFC = listenerConsumerDS.getFeatureSource("listeners")
+
+      val latch = new CountDownLatch(numUpdates)
+
+      val fl = new TestLambdaFeatureListener((fe: KafkaFeatureEvent) => {
+        val f = fe.feature
+        val geom: Point = f.getDefaultGeometry.asInstanceOf[Point]
+        latestLon = geom.getX
+        m.incrementAndGet(f.getID)
+        latch.countDown()
+      })
+
+      consumerFC.addFeatureListener(fl)
+
+      val fw = producerDS.getFeatureWriter("listeners", null, Transaction.AUTO_COMMIT)
+
+      (numUpdates to 1 by -1).foreach { writeUpdate }
+
+      def writeUpdate(i: Int) = {
+        val ll = maxLon - maxLon/i
+
+        val sf = fw.next()
+        sf.getIdentifier.asInstanceOf[FeatureIdImpl].setID("testlistener")
+        sf.setAttributes(Array("smith", 30, DateTime.now().toDate).asInstanceOf[Array[AnyRef]])
+        sf.setDefaultGeometry(gf.createPoint(new Coordinate(ll, ll)))
+        fw.write()
+      }
+
+      logger.info("Wrote feature")
```

Comment:
can we change log to debug, so we don't spam the console during tests? ditto below.